src/Sulu/Bundle/SnippetBundle/Twig/SnippetAreaTwigExtension.php

```diff
@@ -74,6 +83,8 @@ public function loadByArea($area, $webspaceKey = null, $locale = null)
             $locale = $this->requestAnalyzer->getCurrentLocalization()->getLocale();
         }
 
+        $this->snippetAreaReferenceStore->add($area);
```

Comment:
This could let to invalidation of the whole website when a global area snippet is used. Lets discuss this with @chirimoya on thursday.

Reply:
In summary of our discussion, we will introduce a new parameter to the area configuration in the snippet template, which will determine whether the snippet-area should be excluded from the cache invalidation. By default, the snippet-area will always be invalidated. The only exception is for the automatically created `area`, they should not be invalidated, otherwise the old behaviour would be changed.

The decision of whether the invalidation should take place will be integrated into the CacheInvalidationSubscriber. Moreover, the snippet-area will always be included in the reference store. The benefit of this is, that we only have to check in one class if the snippet-area should be invalidated.